gremlin-go/driver/resources/error-messages/en.json

```diff
@@ -0,0 +1,44 @@
+
+  "E0301_GRAPH_GETPATHOBJECT_UNEQUAL_LABELS_OBJECTS_LENGTH_ERROR": "path is invalid because it does not contain an equal number of Labels and Objects",
+  "E0302_GRAPH_GETPATHOBJECT_NON_STRING_VALUE_IN_LABELS_ERROR": "path is invalid because labels contains a non string type",
+  "E0303_GRAPH_GETPATHOBJECT_NO_LABEL_ERROR": "path does not contain a Label of '%s'",
+
+  "E0401_GRAPH_BINARY_WRITETYPEVALUE_UNEXPECTED_NULL_ERROR":"unexpected null value to writeTypeValue when nullable is false",
+  "E0402_GRAPH_BINARY_WRITER_BYTECODE_ERROR": "need GraphTraversal or bytecode to write bytecode",
+  "E0403_GRAPH_BINARY_WRITEVALUE_UNEXPECTED_NULL_ERROR":"unexpected null value to writeValue when nullable is false",
+  "E0404_GRAPH_BINARY_READ_NULLTYPE_ERROR": "expected isNull check to be true for NullType",
+  "E0405_GRAPH_BINARY_READVALUE_NULL_INPUT_ERROR":"input cannot be null",
+  "E0406_GRAPH_BINARY_ENUMREADER_INVALID_TYPE_ERROR": "error, expected string type for enum, but got % x",
+  "E0407_GRAPH_BINARY_GETSERIALIZERTOWRITE_UNKNOWN_TYPE_ERROR":"unknown data type to serialize %s",
+  "E0408_GRAPH_BINARY_GETSERIALIZERTOREAD_UNKNOWN_TYPE_ERROR": "unknown data type to deserialize 0x%x",
+
+  "E0501_PROTOCOL_RESPONSEHANDLER_NO_RESULTSET_ON_DATA_RECEIVE":"resultSet was not created before data was received",
+  "E0502_PROTOCOL_RESPONSEHANDLER_READ_LOOP_ERROR": "error in read loop, error message '%+v'. statusCode: %d",
+  "E0503_PROTOCOL_RESPONSEHANDLER_AUTH_ERROR":"failed to authenticate %v : %v",
+
+  "E0601_RESULT_NOT_VERTEX_ERROR":"result is not a Vertex",
+  "E0602_RESULT_NOT_EDGE_ERROR": "result is not an Edge",
+  "E0603_RESULT_NOT_ELEMENT_ERROR":"result is not an Element",
+  "E0604_RESULT_NOT_PATH_ERROR": "result is not a Path",
+  "E0605_RESULT_NOT_PROPERTY_ERROR":"result is not a Property",
+  "E0606_RESULT_NOT_VERTEX_PROPERTY_ERROR": "result is not a VertexProperty",
+  "E0607_RESULT_NOT_TRAVERSER_ERROR":"result is not a Traverser",
+  "E0608_RESULT_NOT_SLICE_ERROR": "result is not a Slice",
+
+  "E0701_SERIALIZER_READMAP_NULL_KEY_ERROR":"expected non-null Key for map",
+  "E0703_SERIALIZER_READMAP_NON_STRING_KEY_ERROR":"expected string Key for map, got type='0x%x'",
+  "E0704_SERIALIZER_CONVERTARGS_NO_SERIALIZER_ERROR": "failed to find serializer for type %q",
+
+  "E0801_TRANSPORTERFACTORY_GETTRANSPORTLAYER_NO_TYPE_ERROR":"transport layer type was not specified and cannot be initialized",
+
+  "E0901_TRAVERSAL_TOLIST_ANON_TRAVERSAL_ERROR":"cannot invoke this method from an anonymous traversal",
+  "E0902_TRAVERSAL_ITERATE_ANON_TRAVERSAL_ERROR": "cannot invoke this method from an anonymous traversal",
+  "E0903_TRAVERSAL_NEXT_NO_RESULTS_LEFT_ERROR":"there are no results left",
+
+  "E1001_BYTECODE_CHILD_T_NOT_ANON_ERROR": "the child traversal was not spawned anonymously - use the T__ class rather than a TraversalSource to construct the child traversal"
+}
```

Comment:
```suggestion
{
  "E0101_CONNECTION_CLOSE_ERROR": "E0101: cannot close connection that has already been closed or has not been connected",

  "E0201_DRIVER_REMOTE_CONNECTION_CREATESESSION_MULTIPLE_UUIDS_ERROR": "E0201: more than one Session ID specified. Cannot create Session with multiple UUIDs",
  "E0202_DRIVER_REMOTE_CONNECTION_CREATESESSION_SESSION_FROM_SESSION_ERROR": "E0202: connection is already bound to a Session - child sessions are not allowed",

  "E0301_GRAPH_GETPATHOBJECT_UNEQUAL_LABELS_OBJECTS_LENGTH_ERROR": "E0301: path is invalid because it does not contain an equal number of Labels and Objects",
  "E0302_GRAPH_GETPATHOBJECT_NON_STRING_VALUE_IN_LABELS_ERROR": "E0302: path is invalid because labels contains a non string type",
  "E0303_GRAPH_GETPATHOBJECT_NO_LABEL_ERROR": "E0303: path does not contain a Label of '%s'",

  "E0401_GRAPH_BINARY_WRITETYPEVALUE_UNEXPECTED_NULL_ERROR":"E0401: unexpected null value to writeTypeValue when nullable is false",
  "E0402_GRAPH_BINARY_WRITER_BYTECODE_ERROR": "E0402: need GraphTraversal or bytecode to write bytecode",
  "E0403_GRAPH_BINARY_WRITEVALUE_UNEXPECTED_NULL_ERROR":"E0403: unexpected null value to writeValue when nullable is false",
  "E0404_GRAPH_BINARY_READ_NULLTYPE_ERROR": "E0404: expected isNull check to be true for NullType",
  "E0405_GRAPH_BINARY_READVALUE_NULL_INPUT_ERROR":"E0405: input cannot be null",
  "E0406_GRAPH_BINARY_ENUMREADER_INVALID_TYPE_ERROR": "E0406: error, expected string type for enum, but got % x",
  "E0407_GRAPH_BINARY_GETSERIALIZERTOWRITE_UNKNOWN_TYPE_ERROR":"E0407: unknown data type to serialize %s",
  "E0408_GRAPH_BINARY_GETSERIALIZERTOREAD_UNKNOWN_TYPE_ERROR": "E0408: unknown data type to deserialize 0x%x",

  "E0501_PROTOCOL_RESPONSEHANDLER_NO_RESULTSET_ON_DATA_RECEIVE":"E0501: resultSet was not created before data was received",
  "E0502_PROTOCOL_RESPONSEHANDLER_READ_LOOP_ERROR": "E0502: error in read loop, error message '%+v'. statusCode: %d",
  "E0503_PROTOCOL_RESPONSEHANDLER_AUTH_ERROR":"E0503: failed to authenticate %v : %v",

  "E0601_RESULT_NOT_VERTEX_ERROR":"E0601: result is not a Vertex",
  "E0602_RESULT_NOT_EDGE_ERROR": "E0602: result is not an Edge",
  "E0603_RESULT_NOT_ELEMENT_ERROR":"E0603: result is not an Element",
  "E0604_RESULT_NOT_PATH_ERROR": "E0604: result is not a Path",
  "E0605_RESULT_NOT_PROPERTY_ERROR":"E0605: result is not a Property",
  "E0606_RESULT_NOT_VERTEX_PROPERTY_ERROR": "E0606: result is not a VertexProperty",
  "E0607_RESULT_NOT_TRAVERSER_ERROR":"E0607: result is not a Traverser",
  "E0608_RESULT_NOT_SLICE_ERROR": "E0608: result is not a Slice",

  "E0701_SERIALIZER_READMAP_NULL_KEY_ERROR":"E0701: expected non-null Key for map",
  "E0703_SERIALIZER_READMAP_NON_STRING_KEY_ERROR":"E0702: expected string Key for map, got type='0x%x'",
  "E0704_SERIALIZER_CONVERTARGS_NO_SERIALIZER_ERROR": "E0703: failed to find serializer for type %q",

  "E0801_TRANSPORTERFACTORY_GETTRANSPORTLAYER_NO_TYPE_ERROR":"E0801: transport layer type was not specified and cannot be initialized",

  "E0901_TRAVERSAL_TOLIST_ANON_TRAVERSAL_ERROR":"E0901: cannot invoke this method from an anonymous traversal",
  "E0902_TRAVERSAL_ITERATE_ANON_TRAVERSAL_ERROR": "E0902: cannot invoke this method from an anonymous traversal",
  "E0903_TRAVERSAL_NEXT_NO_RESULTS_LEFT_ERROR":"E0903: there are no results left",

  "E1001_BYTECODE_CHILD_T_NOT_ANON_ERROR": "E1001: the child traversal was not spawned anonymously - use the T__ class rather than a TraversalSource to construct the child traversal"
}
```